input/pagecontent/conformance.md

```diff
@@ -1,18 +1,8 @@
-An [AU eRequesting Placer](ActorDefinition-au-erequesting-actor-placer.html):
+Each AU eRequesting actor as defined in [Actors and Capabilities](capability-statements.html):
 - **SHALL** implement the [mandatory](#mandatory-elements) and/or [Must Support](#must-support-and-obligation) requirements for the elements defined as such in the implemented AU eRequesting profiles.
 - **SHALL** implement the required REST API interactions and search parameters as defined in the [AU eRequesting Placer Capability Statement](CapabilityStatement-au-erequesting-placer.html#resourcesSummary1).
```

Comment:
"as defined in the [AU eRequesting Placer Capability Statement](CapabilityStatement-au-erequesting-placer.html#resourcesSummary1)." should be "as defined in their respective CapabilityStatement."

Reply:
Amended, thanks.